lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -157,7 +157,10 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
     // Check for partner embed
     $partner_embed_enabled = dosomething_helpers_get_variable('node', $campaign->nid, 'partner_embed_enable');
     if ($partner_embed_enabled) {
-      $vars['partner_embed_markup'] = dosomething_helpers_get_variable('node', $campaign->nid, 'partner_embed_markup');
+      global $user;
```

Comment:
I know its typically convention we define this at the very top of the function but it felt very disconnected from the purpose it was serving